notebook/static/notebook/js/keyboardmanager.js

```diff
@@ -42,6 +42,46 @@ define([
         this.edit_shortcuts = new keyboard.ShortcutManager(undefined, options.events, this.actions, this.env);
         this.edit_shortcuts.add_shortcuts(this.get_default_common_shortcuts());
         this.edit_shortcuts.add_shortcuts(this.get_default_edit_shortcuts());
+
+
+        this.config = options.config;
+        var that = this;
+
+        this.config.loaded.then(function(){ 
+            var edit_unbind;
+
+            try {
+                edit_unbind = that.config.data.keys.edit.unbind;
+            } catch (e) {
+                if (e instanceof TypeError) {
+                    edit_unbind = [];
+                } else {
+                    throw e;
+                }
+            }
+
+            edit_unbind.forEach(function(u){that.edit_shortcuts.remove_shortcut(u);});
```

Comment:
Yeah, I know. We could think of using es6 transpiler as now we use webpack.... we might be able to use:

`edit_unbind.forEach(that.edit_shortcuts.remove_shortcut, that.edit_shortcuts);` but I'm not sure if:
1) it works, 
2) it's understandable by everyone. 

So I'll be in favor of something obvious.